ruby/private/binary.bzl

```diff
@@ -29,50 +29,27 @@ def _ruby_binary_impl(ctx):
             "main",
         )
 
-    if sdk.is_host:
```

Comment:
Although it is OK to remove this until we really start including runtime into ruby_binary, is it related to rubocop?

Reply:
What ever we do I am not 100% sure we should include the runtime from the toolchain into the binary, and probably should reference another build object, e.g. a ruby_runtime.